docs/src/main/asciidoc/dev-ui.adoc

```diff
@@ -1179,6 +1179,59 @@ export class QwcJokesLog extends LitElement {
 
 https://github.com/phillip-kruger/quarkus-jokes/blob/main/deployment/src/main/resources/dev-ui/qwc-jokes-log.js[Example code]
 
+=== Add a log to the footer
+
+There is an easy way to add a log stream to the footer, without having to create the above mentioned footer. 
+If you just want to stream a log to a tab you can just produce a `FooterLogBuildItem`. This way you only provide a name and a `Flow.Publisher<String>` for your log.
+
+Here is an example from the Dev Services deployment module:
+
+[source,java]
+----
+@BuildStep(onlyIf = { IsDevelopment.class })
+public List<DevServiceDescriptionBuildItem> config(
+        // ...
+        BuildProducer<FooterLogBuildItem> footerLogProducer){
+
+        // ...
+
+    // Dev UI Log stream
+    for (DevServiceDescriptionBuildItem service : serviceDescriptions) {
+        if (service.getContainerInfo() != null) {
+            footerLogProducer.produce(new FooterLogBuildItem(service.getName(), () -> {
+                return createLogPublisher(service.getContainerInfo().getId());
+            }));
+        }
+    }
+}
+
+// ...
+
+private Flow.Publisher<String> createLogPublisher(String containerId) {
```

Comment:
Can we explan what this is doing? I have 0️⃣ understanding just by eye-balling it for the first time